algorithms/strings/remove_duplicate.py

```diff
@@ -0,0 +1,11 @@
+def remove_dups(string):
```

Comment:
the method name looks a bit misleading. If I understand correctly, the program removes adjacent duplicates (rather than removing duplicates)
Input: `faacaacccgfs`
Output: `facacgfs`

Reply:
yeah, right.. you got that.